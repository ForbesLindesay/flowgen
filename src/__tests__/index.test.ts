import {transformFile} from '../';

test('index.example.ts', () => {
  expect(transformFile(__dirname + '/index.example.d.ts')).toMatchSnapshot();
})