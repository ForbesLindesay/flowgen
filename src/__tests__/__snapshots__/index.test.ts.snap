// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index.example.ts 1`] = `
"// @flow
// Generated using flowgen2

import type {Foo} from './index-headers.example';
import type {Bar} from './index-headers.example';
export type {Foo};
export type {Bar as Baz};
"
`;
